catalog/ui/catalog-ui-search/package.json

```diff
@@ -70,8 +70,8 @@
     "webpack-merge": "0.14.0"
   },
   "dependencies": {
-    "@turf/circle": "3.3.3",
```

Comment:
I am concerned about potential regressions from lowering the turf version.  How was this regression tested?

It looks like public scoped packages should work with Yarn.
https://github.com/yarnpkg/yarn/issues/825

Maybe we can try adding the namespace to the npmrc file for this project.
https://github.com/yarnpkg/yarn/pull/839
`@turf:registry=http://registry.npmjs.org/`